ballerina-shell/modules/shell-core/src/test/java/io/ballerina/shell/test/unit/TrialTreeParserTest.java

```diff
@@ -90,4 +102,12 @@ private void testParse(String fileName, Class<?> parentClazz) {
             }
         }
     }
+
+    private void testModuleDclnName(String fileName) throws TreeParserException {
+        TestCases testCases = TestUtils.loadTestCases(fileName, TestCases.class);
+        TreeParser treeParser = TestUtils.getTestTreeParser();
+        for (TestCase testCase : testCases) {
+                treeParser.parse(testCase.getInput());
```

Comment:
slight formatting issue here

Reply:
Fixed in 5732df29120212bc9c5639fb1901b98d6172d290